deployment/ccip/changeset/cs_add_chain_test.go

```diff
@@ -30,6 +30,7 @@ import (
 )
 
 func TestAddChainInbound(t *testing.T) {
+	t.Skipf("Skipping test as it is running into timeout issues, move the test into integration in-memory tests")
```

Comment:
Can you create a ticket for this? Its weird that its timing out, it takes 90s on my laptop

Reply:
Yes created https://smartcontract-it.atlassian.net/browse/CCIP-4594